packages/request-manager/e2e/torRunner.ts

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
+import { spawn } from 'child_process';
+
+const customTorProcessDir = process.env.TOR_BINARY_PATH;
+const processDir = path.join(__dirname, './../../suite-data/files/bin/tor/linux-x64/');
+const processPath = customTorProcessDir || path.join(processDir, 'tor');
+
+export const torRunner = ({ torParams = [] }: { torParams: string[] }) => {
+    console.log('torParams', torParams);
+    const process = spawn(processPath, torParams, {
+        cwd: processDir,
+        stdio: ['ignore', 'ignore', 'ignore'],
+    });
+    process.on('error', err => {
+        console.log('err', err);
+    });
+    process.on('exit', code => {
+        console.log('exit code', code);
```

Comment:
this throws errors after tests:
```
Cannot log after tests are done. Did you forget to wait for something async in your test?
```

Reply:
Yes, that is not necessary 5b2869c4e625edbcbcbbc7c53ac0207dea7f2504